test/conformance/configuration_test.go

```diff
@@ -34,7 +34,7 @@ func TestUpdateConfigurationMetadata(t *testing.T) {
 	logger := logging.GetContextLogger(t.Name())
 
 	names := test.ResourceNames{
-		Config: test.AppendRandomString("test-update-configuration-meta-", logger),
+		Config: test.AppendRandomString("test-update-configuration-meta-"),
```

Comment:
The new one appends the `-` itself. So we probably can remove `-` here and elsewhere, or just live with double dashes.

Reply:
cc @tcnghia  who wants to improve the names we use everywhere too.